src/index.css

```diff
@@ -17,6 +17,9 @@
   --absent-cell-bg-color: theme('colors.slate.400');
   --correct-cell-bg-color: theme('colors.green.400');
   --present-cell-bg-color: theme('colors.yellow.400');
+  --wrong-char-cell-bg-color: #af55e0;
```

Comment:
I would rather keep the tailwind convention here and use theme colors instead of hardcoding hex values.